src/parse.rs

```diff
@@ -84,6 +84,14 @@ impl Request {
             ..self
         }
     }
+
+    pub(crate) fn ends_with_slash(&self) -> bool {
+        if cfg!(target_os = "windows") {
+            self.target.ends_with('\\')
+        } else {
+            self.target.ends_with('/')
+        }
+    }
```

Comment:
the target comes from the user, this should probably be `self.target.ends_with('/') || self.target.ends_with('\\')`?

Reply:
maybe we only need `self.target.ends_with('/')`

Reply:
enhanced-resolve only uses  `self.target.ends_with('/')`